packages/strapi-design-system/src/Avatar/Avatar.js

```diff
@@ -1,21 +1,66 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
 
-const avatarSize = '26px';
+const avatarSize = 26;
+const previewSize = 64;
 
 const AvatarImg = styled.img`
   border-radius: 50%;
   display: block;
-  height: ${avatarSize};
-  width: ${avatarSize};
+  position: relative;
 `;
 
-export const Avatar = ({ src, alt }) => {
-  return <AvatarImg src={src} alt={alt} width={avatarSize} height={avatarSize} />;
+const AvatarImgWrapper = styled.div`
+  position: relative;
+`;
+
+const PreviewContainer = styled.img`
+  border-radius: 50%;
+  position: absolute;
+  transform: translate(-${(previewSize - avatarSize) / 2}px, -100%);
+  margin-top: -${({ theme }) => theme.spaces[1]};
```

Comment:
Maybe we will need to add some zIndex here wdyt ? 🤔

Reply:
do we need a z-index? 🤔

Reply:
Probably not, we'll see on the use 👍🏽